cli/types/cypress.d.ts

```diff
@@ -107,6 +107,11 @@ declare namespace Cypress {
     fromAutWindow: WindowPosition & { x: number, y: number }
   }
 
+  /**
+   * The interface for user-defined properties in Window object under test.
+   */
+  interface ApplicationWindow {} // tslint:disable-line
```

Comment:
can tslint disable specific error? Like in this case it is probably an empty interface error, right?

Reply:
I don't remember the message, but tslint doesn't allow empty interface(`{}`). 

Unfortunately, tslint doesn't have a way to disable only a rule like `/* eslint-disable a-rule */`.